libs/@blockprotocol/graph/src/stdlib/subgraph/builder.ts

```diff
@@ -48,7 +48,7 @@ export const buildSubgraph = (
       ),
   );
 
-  if (missingRoots) {
+  if (missingRoots.length > 0) {
```

Comment:
Bug fix: `missingRoots` is the result of a `filter` (this was left over from when it was a `find`)